packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs

```diff
@@ -30,25 +30,33 @@ impl<C> Platform<C> {
         // if we are at an epoch change, check to see if over 75% of blocks of previous epoch
         // were on the future version
         let protocol_versions_counter = self.drive.cache.protocol_versions_counter.read();
+
         let mut versions_passing_threshold =
             protocol_versions_counter.versions_passing_threshold(required_upgraded_hpmns);
-        drop(protocol_versions_counter);
 
         if versions_passing_threshold.len() > 1 {
+            drop(protocol_versions_counter);
```

Comment:
This variable will be dropped on the next line. I don't think you need to do explicitly.